tests/test_client.py

```diff
@@ -157,20 +159,6 @@ def test_find_retractions_of_valid_only(self):
         unexpected_uri = 'http://purl.org/np/RACdYpR-6DZnT6JkEr1ItoYYXMAILjOhDqDZsMVO8EBZI'
         assert unexpected_uri not in results
 
-    def test_nanopub_search(self):
```

Comment:
I removed this, because I think it is redundant.

Reply:
Agreed.